### PR TITLE
Dice fixes

### PR DESCRIPTION
**Pull request by GrigorenkoPV**
1. Fixed a typo in [org.thunderdog.challegram.telegram.Tdlib](https://github.com/TGX-Android/Telegram-X/compare/main...GrigorenkoPV:dice?expand=1#diff-4fe065d18d907c74daead3160725d650603dc0216cf5c57ef0f2b2d708b9c842) which caused dice and similar emojis not being recognized as Telegram's special interactive emojis. 
2. If you open a context menu for a message with a dice, there is a special "Send a dice" button. Pressing it indeed sends a dice. However, this used to send this dice without replying to any messages, regardless of whether the user was writing a reply or not. This has been fixed.
3. The change above allowed to inline some constant parameters of `sendContent` method and simplify the code a bit. Furthermore, I have taken the liberty to remove the special treatment for `.webp` files, which would send them as stickers instead of documents. This does not seem to cause any issues, and even allows to get rid of a single-use `sendSticker` overload together with a special overload for `sendContent`.

Not sure if I should squash my commits or not, so I will leave them this way for now. Anyways, thanks in advance for reviewing this PR! ❤️

### CODE REVIEW

**Review by vkryl — CHANGES_REQUESTED** [review]
Quote from [PR guide](https://github.com/TGX-Android/Telegram-X/blob/main/docs/PULL_REQUEST_TEMPLATE.md#when-fixing-issues-make-sure-that-your-pr-is):

> **Separated.** Different bug fixes are divided into independent PRs.

While the typo fix related to sending dice message is clearly acceptable, changes made to the `MessagesController.java` are controversial and do not seem to be related to the dice messages.

I suggest reverting `MessagesController.java` in this PR, and keeping only the bugfix-related change(s).

File: app/src/main/java/org/thunderdog/challegram/ui/MessagesController.java [hunk 1]
```diff
@@ -9391,9 +9383,7 @@ public void onActivityResult (int requestCode, int resultCode, Intent data) {
           return;
         }
 
-        if (imagePath != null && imagePath.endsWith(".webp")) {
-          sendSticker(imagePath, true, Td.newSendOptions());
-        } else if (requestCode == Intents.ACTIVITY_RESULT_GALLERY_FILE) {
```
[the remark this answers is not in the document]
Comment (GrigorenkoPV) [reply app/src/main/java/org/thunderdog/challegram/ui/MessagesController.java]: What is the reason for sending `.webp` files as stickers?

**Comment by GrigorenkoPV** [comment]
Thanks for the feedback!

I have force-pushed a new set of commits for this PR. It is much smaller and does not contain any controversial or non-dice-related changes now.

To be more precise, now it has these and only these two changes:

> 1. Fixed a typo in [org.thunderdog.challegram.telegram.Tdlib](https://github.com/TGX-Android/Telegram-X/pull/331/files#diff-4fe065d18d907c74daead3160725d650603dc0216cf5c57ef0f2b2d708b9c842) which caused dice and similar emojis not being recognized as Telegram's special interactive emojis.
> 2. If you open a context menu for a message with a dice, there is a special "Send a dice" button. Pressing it indeed sends a dice. However, this used to send this dice without replying to any messages, regardless of whether the user was writing a reply or not. This has been fixed.

**Review by vkryl — CHANGES_REQUESTED** [review]
Requesting to revert `MessagesController.java` to limit this PR to inability to properly detect dice emoji.

**Comment by GrigorenkoPV** [comment]
> Requesting to revert `MessagesController.java` to limit this PR to inability to properly detect dice emoji.

Done.

**Review by vkryl — APPROVED** [review]

**Pull request merged**